qiling/extensions/tracing/formats/tenet.py

```diff
@@ -0,0 +1,103 @@
+# This code structure is copied and modified from the coverage extension
+
+import qiling
```

Comment:
Better do `from qiling import Qiling`.
If this is only for type annotations, then it's would be best to put it under `if TYPE_CHECKING` to reduce unnecesary runtime depedencies.